### ISSUE TITLE
Large Display: Cannot toggle Share App Usage

### ISSUE THREAD

**Issue by denisedresbach** [covid-alert-ny/covid-green-app#565]
App version: iOS 1.1.0.52
OS: iOS 14
Device: iPhone SE
Reproducible: Always

Summary: Large Display: Cannot toggle Share App Usage

1. Set Accessibility Display size to Largest
2. Launch app and go to Settings >  Help Improve This App
3. Scroll to bottom of screen

Results: Switch is off the screen some and it is very hard to toggle it

![IMG_0371](https://user-images.githubusercontent.com/69995161/97629055-c4d0ec00-19ea-11eb-9e99-251705430e2a.PNG)

**Comment by AlanSl** [comment]
Should be fixed in build 53

**Comment by denisedresbach** [comment]
Verified fixed 1.1.0.53
